app/components/SortableMenuItem.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import type { MenuItemProps } from 'semantic-ui-react';
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+
+import { Menu } from './basic';
+
+export default function SortableMenuItem(props: MenuItemProps) {
+    const { id, ...rest } = props;
```

Comment:
Minor thing - maybe shorter version?:
```suggestion
export default function SortableMenuItem({ id, ...rest }: MenuItemProps) {
```